src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -164,6 +164,11 @@ public function load(array $configs, ContainerBuilder $container)
             $definition->replaceArgument(1, null);
         }
 
+        $this->addAnnotatedClassesToCache(array(
+            'Bundle\\Controller\\',
```

Comment:
I think it would be more explicit here to do the same as composer: use `*` to match anything but `\`, and `**` to match any char (and have `**` be implicit at the end) . Then replace this with `'**Bundle\\Controller\\'`.